### ISSUE TITLE
Engine Panel JFS Starter Ready Light not working

### ISSUE THREAD

**Issue by rocketeer998** [DCS-Skunkworks/dcs-bios#528]
**DCS-BIOS version**
v7.50. Downloaded yesterday 2/15/24

**DCS-BIOS Control**
F-15E Engine Panel
LED fGenStarterL

**Describe the bug**
when you press the light test toggle in the Internal Light Panel, the JFS starter light comes on in DCS. but physical switch does not light up.

I tried the old and new code.
DcsBios::LED fGenStarterL(0x93f0, 0x0100, PIN);

DcsBios::LED fGenStarterL(F_15E_F_GEN_STARTER_L_AM, PIN);

Both don't work.

**Comment by jdahlblom** [comment]
Verified this is working with:
* ```DCS-BIOS Nightly Build```
* ```DCSBIOSBridge```
* Monitored the ```DCS-BIOS``` value for ```F_GEN_STARTER_L``` using ```BIOSBuddy```

```DcsBios::LED fGenStarterL(F_15E_F_GEN_STARTER_L_AM, greenLed);```

You have something wrong with your setup.

**Comment by rocketeer998** [comment]
old 0x93f0 didn't work.

new code F_15E_F_GEN_STARTER_L_AM now cannot even compile in Arduino.
![bios error2](https://github.com/DCS-Skunkworks/dcs-bios/assets/145094080/6644b36a-bf87-4303-b458-8ab09218a1c8)


once again, grabbing this from Bort works when I press the test light toggle, it lights up in the game and on my light indicator.
DcsBios::LED fGenStarterL(0x93e4, 0x0100, PIN);

So this engine JFS light, light the CAS takeoff trim button light, both worked from the code from Bort in the game, not from the code you declared working above from the DCSBiosBridge, both can't even compile in Arduino.

Appreciate you guys providing support for dcs bios. But you don't want to test it in the game itself, which is the main purpose of gamers buying the Saitek/Logitech flight panels and use them for DCS aircrafts, besides civilian flying in MSFS. I have proven twice that the code from Bort is the one that works, at least for LED, not the one you declared working, which I could not even compile. We the users have to compile the given code into Arduino to make the switches work. So when it didn't work for me, you should not just say something wrong with my setup when the code you provide can't even be compiled for upload thus can't be used in the game.

**Comment by charliefoxtwo** [comment]
@maciekish any idea why this isn't compiling?

**Comment by jdahlblom** [comment]
```userguide.html```:

![macros](https://github.com/DCS-Skunkworks/dcs-bios/assets/10453261/7e5d47e0-e8a0-45c3-8d10-7e12db28de18)

**Comment by jdahlblom** [comment]
Problem solved, closing this issue.

**Comment by maciekish** [comment]
> @maciekish any idea why this isn't compiling?

Sorry was busy at work. Either an old or missing Addresses.h. We need to make sure the file is up to date and included everywhere.

**Comment by rocketeer998** [comment]
old code
DcsBios::LED fGenStarterL(0x93f0, 0x0100, 43);

new code that can't compile
DcsBios::LED fGenStarterL(F_15E_F_GEN_STARTER_L_AM, 43);

latest code from Bort
DcsBios::LED fGenStarterL(0x93e4, 0x0100, 43);

I noticed you guys changed all LED code throughout the same cockpit from 0xNNNN to names only and they either don't work or don't compile. Looking at the error it was suggesting some libraries missing something. If you guys have fixed it by adding whatever was missing in address.h etc that's good. Else people using code in dcs bios download will still get error. For me, I have found that the old format 0xNNNN from Bort works and I've been doing that since. But often the code found in Bort is different from even the latest version of dcs bios download. Many people still don't know about Bort so they won't try to use Bort's code when they get stuck with the code given in dcs bios download control-reference.html.
